frontend/test/metabase/scenarios/dashboard/dashboard-drill.cy.spec.js

```diff
@@ -401,6 +401,109 @@ describe("scenarios > dashboard > dashboard drill", () => {
+          ],
+        },
+        type: "query",
+      },
+      display: "bar",
+      visualization_settings: {},
+    }).then(({ body: { id: QUESTION_ID } }) => {
+      cy.log("**-- 2. Create a dashboard --**");
+
+      cy.request("POST", "/api/dashboard", {
+        name: "13785D",
+      }).then(({ body: { id: DASHBOARD_ID } }) => {
+        cy.log("**-- 3. Add filter to the dashboard --**");
+
+        cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}`, {
+          parameters: [
+            {
+              id: "4ff53514",
+              name: "Date Filter",
+              slug: "date_filter",
+              type: "date/all-options",
+            },
+          ],
+        });
+        cy.log("**-- 4. Add question to the dashboard --**");
+
+        cy.request("POST", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+          cardId: QUESTION_ID,
+        }).then(({ body: { id: DASH_CARD_ID } }) => {
+          cy.log("**-- 5. Connect dashboard filter to the question --**");
+
+          cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+            cards: [
+              {
+                id: DASH_CARD_ID,
+                card_id: QUESTION_ID,
+                row: 0,
+                col: 0,
+                sizeX: 14,
+                sizeY: 10,
+                series: [],
+                // Set "Click behavior"
+                visualization_settings: {
+                  click_behavior: {
+                    type: "crossfilter",
+                    parameterMapping: {
+                      "4ff53514": {
+                        source: {
+                          type: "column",
+                          id: "CREATED_AT",
+                          name: "Created At",
+                        },
+                        target: {
+                          type: "parameter",
+                          id: "4ff53514",
+                        },
+                        id: "4ff53514",
+                      },
+                    },
+                  },
+                },
+                // Connect filter and card
+                parameter_mappings: [
+                  {
+                    parameter_id: "4ff53514",
+                    card_id: QUESTION_ID,
+                    target: ["dimension", ["field-id", REVIEWS.CREATED_AT]],
+                  },
+                ],
+              },
+            ],
+          });
+        });
+        cy.server();
+        cy.route("POST", `/api/card/${QUESTION_ID}/query`).as("cardQuery");
+
+        cy.visit(`/dashboard/${DASHBOARD_ID}`);
+
+        cy.wait("@cardQuery");
+        cy.get(".bar")
+          .eq(14) // August 2017 (Total of 12 reviews, 9 unique days)
+          .click({ force: true });
+
+        cy.wait("@cardQuery.2");
+        cy.url().should("include", "2017-08-01~2017-08-31");
+        // For this to work, binning would have to change to "group by day"
+        cy.get(".bar").should("have.length", 9);
```

Comment:
I would not expect binning to be changed, since it's not a drill-through. That's likely feature request 5346/6583.
But I would definitely expect it to return the count for the entire period - count=12

Reply:
So, should I assert only on a filter (range) change for now? That's going to be reflected in the URL, and there is already an assertion in place for that.

When you say
> I would definitely expect it to return the count for the entire period - count=12

Do you mean we already have such mechanism in place or do you expect this to behave that way after the fix?

Reply:
I mean if you set the filter to the entire month, then there should only be a single bar, since it's grouped by month, and the value should be 12. That would be correct. Changing the breakout/etc would be different feature requests that this issue.

Reply:
Ok, one bar. Noted.
Will filter range change? In other words, is this part correct?
`cy.url().should("include", "2017-08-01~2017-08-31");`

Reply:
Yes, and only a single bar with the value 12.
![image](https://user-images.githubusercontent.com/1447303/106482905-3d45be00-64ae-11eb-88d2-f8f4bc6fcfb0.png)